Core/Validation/UserRegistrationValidation.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Core.DTO.UserDTO;
+using Core.Entities.UserEntity;
+using FluentValidation;
+using Microsoft.AspNetCore.Identity;
+
+namespace Core.Validation
+{
+    public class UserRegistrationValidation : AbstractValidator<UserRegistrationDTO>
+    {
+        protected readonly UserManager<User> _userManager;
+
+        public UserRegistrationValidation(UserManager<User> userManager)
+        {
+            _userManager = userManager;
+
+            RuleFor(user => user.Name)
+                .NotNull()
+                .NotEmpty()
+                .Length(2, 50)
+                .Matches(@"^\D*$").WithMessage("'{PropertyName}' must not contain digits!")
+                .Matches(@"\A[A-Z][a-z]{1,49}[a-z]*$").WithMessage("The first letter in '{PropertyName}' must be uppercase, the others must be lowercase!")
+                .Matches("^[^£# “”!@$%^&*(){}:;<>,.?/+_=|'~\\-]*$").WithMessage("'{PropertyName}' must not contain special characters!");
+
+            RuleFor(user => user.Surname)
+                .NotNull()
+                .NotEmpty()
+                .Length(2, 50)
+                .Matches(@"^\D*$").WithMessage("'{PropertyName}' must not contain digits!")
+                .Matches(@"\A[A-Z][a-z]{1,49}[a-z]*$").WithMessage("The first letter in '{PropertyName}' must be uppercase, the others must be lowercase!")
+                .Matches("^[^£# “”!@$%^&*(){}:;<>,.?/+_=|'~\\-]*$").WithMessage("'{PropertyName}' must not contain special characters!");
```

Comment:
Investigate ways to avoid RegExp duplication in the rules.

Reply:
We will fix this problem in the next sprint